notebooks/inference_SavedModel_VertexEndpoint.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<p>This prediction result is not matching with the one we showed above. This shouldn't be the case. No?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/semantic-segmentation-ml-pipeline/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>Hm.. this is weird result. I am using the same model. The only difference is whether hosted on local or Vertex Endpoint. </p>